docs-ref-services/latest/resourcemanager-msi-readme.md

```diff
@@ -6,7 +6,8 @@ ms.author: weidxu
 ms.date: 06/24/2022
 ms.topic: reference
 ms.devlang: java
-ms.service: azure-arc-data
+ms.service: active-directory
```

Comment:
should the keywords on line 3 also be updated?

Reply:
I just saw "\managed-service-identity.md" changed to  ms.service: managedserviceidentity
I should change "\resourcemanager-msi-readme.md" to the same value?

Reply:
@soma-ms

I believe this file is updated by doc stage in SDK release CI.

Does the change in PR stays, or do we need to modify another configure file?

Reply:
@weidongxu-microsoft - I don't have context on this repository. You probably want to tag someone else here.

Reply:
This is not the place to change ms.service.
Please submit PR in https://github.com/Azure/azure-sdk/blob/main/_data/releases/latest/java-packages.csv
The `MSDocService` attribute is for updating the value of `ms.service` here.

Reply:
Thanks all. I'll close this PR without merging